src/main/webapp/js/feedbackSubmissionsEdit.es6

```diff
@@ -15,8 +15,10 @@ const FEEDBACK_RESPONSE_RECIPIENT = 'responserecipient';
 const FEEDBACK_RESPONSE_TEXT = 'responsetext';
 const FEEDBACK_MISSING_RECIPIENT = 'You did not specify a recipient for your response in question(s)';
 const WARNING_STATUS_MESSAGE = '.alert-warning.statusMessage';
+const WARNING_STATUS_MESSAGE_SESSION_CLOSE = 'You have only 15 minutes left to finish this feedback session. Hurry up!.';
```

Comment:
Use `Warning: you have only x minutes before the submission deadline expires!` as per https://github.com/TEAMMATES/teammates/issues/1872#issuecomment-296169944